Testscripts/Linux/SetupRDMA.sh

```diff
@@ -292,6 +327,9 @@ function Main() {
 
 		# add Intel MPI path to PATH
 		export PATH=$PATH:"${mpirun_path%/*}"
+		# add sourcing file in each session
+		echo "source ${mpirun_path%/*}/mpivars.sh" >> $HOMEDIR/.bashrc
+		echo "source ${mpirun_path%/*}/mpivars.sh" >> /home/lisa/.bashrc
```

Comment:
hardcode user lisa is not recommended.

Reply:
Fixed in d560a58210af8358549b783f42f2e5dfa49333ec